src/Api/Models/Response/Organizations/OrganizationDomainResponseModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.Models.Response.Organizations;
+
+public class OrganizationDomainResponseModel : ResponseModel
+{
+    public OrganizationDomainResponseModel(OrganizationDomain organizationDomain, string obj = "organizationDomain")
+        : base(obj)
+    {
+        if (organizationDomain == null)
+        {
+            throw new ArgumentNullException(nameof(organizationDomain));
+        }
+
+        Id = organizationDomain.Id.ToString();
+        OrganizationId = organizationDomain.OrganizationId.ToString();
+        Txt = organizationDomain.Txt;
+        DomainName = organizationDomain.DomainName;
+        CreationDate = organizationDomain.CreationDate;
+        NextRunDate = organizationDomain.NextRunDate;
+        NextRunCount = organizationDomain.NextRunCount;
+        VerifiedDate = organizationDomain.VerifiedDate;
+    }
+
+    public string Id { get; set; }
+    public string OrganizationId { get; set; }
+    public string Txt { get; set; }
```

Comment:
What is this `Txt` property? Is it the text file added to a domain or its location?

Reply:
`Txt` is a type of record on the `DNS` record. A random string would be generated from our end, which the Organization has to input on their `TXT` record on their DNS.

Reply:
Got it, thanks @gbubemismith !